doc/classes/Control.xml

```diff
@@ -133,6 +133,14 @@
 				Return [code]Array[/code] of [code]Vector2i[/code] text ranges, in the left-to-right order. Ranges should cover full source [code]text[/code] without overlaps. BiDi algorithm will be used on each range separately.
 			</description>
 		</method>
+		<method name="_unhandled_key_input" qualifiers="virtual">
```

Comment:
That's already bound in `Node`, why is it now re-bound in `Control` (and with wrong type hint)?

Comes from efe5c250d5c @EricEzaM.

Reply:
First guess would be that I didn't bind it as virtual? In mobile can't check atm

Reply:
Why does it need to be bound though? It's already bound in Node, and Control inherits Node.

Reply:
Yeah that too lol

Reply:
I'll fix it up when I get to my computer

Reply:
Fixed.